tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -2636,10 +2663,16 @@ def test_OnDiskDataset_homogeneous(include_original_edge_id, edge_fmt):
         with open(yaml_file, "w") as f:
             f.write(yaml_content)
 
-        dataset = gb.OnDiskDataset(
-            test_dir, include_original_edge_id=include_original_edge_id
-        ).load()
+        if include_original_edge_id:
```

Comment:
I think it's ok to wrap directly instead of `if-else`

Reply:
DONE